crowdstrikeoauthapi_connector.py

```diff
@@ -28,7 +29,7 @@
 import phantom.utils as util
 import pytz
 import requests
-import simplejson as json
+# import simplejson as json
```

Comment:
We can remove unnecessary imports instead of commenting out.

Reply:
Done